### PR TITLE
[LC-850] remove peer_manager in all code.

### PR DESCRIPTION
**Pull request by windies21**
* There is IISS, so no peer_manager is required.

### CODE REVIEW

File: loopchain/blockchain/blockchain.py [hunk 1]
```diff
@@ -1078,6 +1090,13 @@ def _init_blockchain(self):
             logging.debug("restore from last block hash(" + str(self.__last_block.header.hash.hex()) + ")")
             logging.debug("restore from last block height(" + str(self.__last_block.header.height) + ")")
 
+    def init_crep_reps(self) -> None:
```
Comment (metalg0su) [inline loopchain/blockchain/blockchain.py]: I think this method aims `init_reps_hash`.

File: loopchain/channel/channel_service.py [hunk 2]
```diff
@@ -22,17 +22,18 @@
 
 from loopchain import configure as conf
 from loopchain import utils
-from loopchain.baseservice import (BroadcastScheduler, BroadcastSchedulerFactory, ObjectManager, CommonSubprocess,
+from loopchain.baseservice import (BroadcastScheduler, BroadcastSchedulerFactory, ObjectManager,
+                                   CommonSubprocess,
```
Comment (metalg0su) [inline loopchain/channel/channel_service.py]: Any reason to break line for `CommonSubprocess`?

File: loopchain/blockchain/blockchain.py [hunk 3]
```diff
@@ -137,11 +139,22 @@ def _keep_order_in_penalty(self) -> bool:
...
+            if self.__last_block.header.prep_changed_reason == NextRepsChangeReason.NoChange:
+                utils.logger.debug(f"There is no change in reps.")
+                return
+
+            new_reps = self.find_preps_addresses_by_roothash(self.__last_block.header.revealed_next_reps_hash)
+            new_node_type = NodeType.CommunityNode if ChannelProperty().peer_address in new_reps else NodeType.CitizenNode
+            is_switched_role = new_node_type != ChannelProperty().node_type
+        else:
+            is_switched_role = False
+
```
Comment (metalg0su) [inline loopchain/blockchain/blockchain.py]: I can see those lines has been migrated partially from `PeerManager.reset_all_peers`.
But I cannot understand what those lines do.

Please consider:
- Restore signature of `reset_leader_made_block_count` and its contents. 
- Extract those lines as `is_role_switched` or something reasonable, then pass its return value to `reset_leader_made_block_count`. (I think this method does more than one thing, which makes me confused.)

File: loopchain/channel/channel_service.py [hunk 4]
```diff
@@ -259,7 +256,7 @@ def _is_role_switched(self) -> bool:
         self.__inner_service.update_sub_services_properties(node_type=ChannelProperty().node_type.value)
 
     def switch_role(self):
-        self.peer_manager.update_all_peers()
+        self.__block_manager.blockchain.reset_leader_made_block_count(need_check_switched_role=True)
```
Comment (metalg0su) [inline loopchain/channel/channel_service.py]: Related: https://github.com/icon-project/loopchain/pull/462/files#r369901917

**Review by metalg0su — DISMISSED** [review]

File: loopchain/blockchain/blockchain.py (hunk 1 above)
Comment (windies21) [reply loopchain/blockchain/blockchain.py]: Well.... this is only for origin CREPs.

File: loopchain/channel/channel_service.py (hunk 2 above)
Comment (windies21) [reply loopchain/channel/channel_service.py]: Haha my editor hard wrap is 110. It allows editing three pages on one screen.

**Review by metalg0su — APPROVED** [review]

**Pull request merged**